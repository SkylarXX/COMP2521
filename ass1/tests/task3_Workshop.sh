./task3 tests/Workshop
