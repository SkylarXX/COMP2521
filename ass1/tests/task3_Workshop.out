/----------
| M-ID: <130F095F-8328-4AE7-A04E-400BCC65ECBF@cse.unsw.edu.au>
| Subj: Managing teaching with less casual assistance
| From: John Shepherd <jas@cse.unsw.edu.au>
| Date: Sun Nov 13 12:29:05 2011
| Repl: (null)
\----------
   /----------
   | M-ID: <B602879E-CD3F-4FCD-B3C1-51243092209C@cse.unsw.edu.au>
   | Subj: Re: Managing teaching with less casual assistance
   | From: Maurice Pagnucco <mp@cse.unsw.edu.au>
   | Date: Mon Nov 14 23:41:42 2011
   | Repl: <130F095F-8328-4AE7-A04E-400BCC65ECBF@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <299F345B-B944-4C87-884C-52E2F493850F@cse.unsw.edu.au>
      | Subj: Re: Managing teaching with less casual assistance
      | From: John Shepherd <jas@cse.unsw.edu.au>
      | Date: Tue Nov 15 10:40:06 2011
      | Repl: <B602879E-CD3F-4FCD-B3C1-51243092209C@cse.unsw.edu.au>
      \----------
/----------
| M-ID: <28052D39-2D48-4A92-B521-65C8BEBD7EB5@cse.unsw.edu.au>
| Subj: Teaching practices workshop
| From: Maurice Pagnucco <mp@cse.unsw.edu.au>
| Date: Fri Nov 18 00:46:06 2011
| Repl: (null)
\----------
   /----------
   | M-ID: <F3BB7E77-37BA-49CA-8853-C98CEEEA4050@cse.unsw.edu.au>
   | Subj: Re: Teaching practices workshop
   | From: "Richard Buckland (CSE@UNSW)" <rb@cse.unsw.edu.au>
   | Date: Fri Nov 18 03:54:10 2011
   | Repl: <28052D39-2D48-4A92-B521-65C8BEBD7EB5@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <5FABA40C-0352-4DF4-8AC8-155DEEC94887@cse.unsw.edu.au>
      | Subj: Re: Teaching practices workshop
      | From: John Shepherd <jas@cse.unsw.edu.au>
      | Date: Fri Nov 18 10:35:10 2011
      | Repl: <F3BB7E77-37BA-49CA-8853-C98CEEEA4050@cse.unsw.edu.au>
      \----------
         /----------
         | M-ID: <CALQ3V4KeRfKrm1jP0EtPM5+vNRPe8JtY2jUUfM=EPRQ2Up7LEg@mail.gmail.com>
         | Subj: Re: Teaching practices workshop
         | From: Ashesh Mahidadia <am@cse.unsw.edu.au>
         | Date: Fri Nov 18 10:44:53 2011
         | Repl: <5FABA40C-0352-4DF4-8AC8-155DEEC94887@cse.unsw.edu.au>
         \----------
      /----------
      | M-ID: <C3A06CDD-4D47-419D-B6E9-1308BE28E11D@cse.unsw.edu.au>
      | Subj: Re: Teaching practices workshop
      | From: John Shepherd <jas@cse.unsw.edu.au>
      | Date: Fri Nov 18 11:46:37 2011
      | Repl: <F3BB7E77-37BA-49CA-8853-C98CEEEA4050@cse.unsw.edu.au>
      \----------
         /----------
         | M-ID: <AE66195C-639B-4E4B-931E-7C139A220CE8@cse.unsw.edu.au>
         | Subj: Re: Teaching practices workshop
         | From: Eric Martin <em@cse.unsw.edu.au>
         | Date: Fri Nov 18 11:59:11 2011
         | Repl: <C3A06CDD-4D47-419D-B6E9-1308BE28E11D@cse.unsw.edu.au>
         \----------
         /----------
         | M-ID: <CALQ3V4KJ30ith8h68+d3P_tLXf2GMax5PkxKf8NGO=M9nwp-3w@mail.gmail.com>
         | Subj: Re: Teaching practices workshop
         | From: Ashesh Mahidadia <am@cse.unsw.edu.au>
         | Date: Fri Nov 18 12:06:22 2011
         | Repl: <C3A06CDD-4D47-419D-B6E9-1308BE28E11D@cse.unsw.edu.au>
         \----------
/----------
| M-ID: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
| Subj: Workshop on Teaching with Reduced Casual Assistance
| From: John Shepherd <jas@cse.unsw.edu.au>
| Date: Mon Nov 28 12:50:26 2011
| Repl: (null)
\----------
   /----------
   | M-ID: <CA+uZbSofVTVfqf3ULohiPvx8R2Kwn=urZpqxeZLxtbZUR55UZw@mail.gmail.com>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Srikumar Venugopal <sv@cse.unsw.edu.au>
   | Date: Mon Nov 28 13:00:01 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <4ED2DF07.4010207@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Oliver Diessel <odiessel@cse.unsw.edu.au>
   | Date: Mon Nov 28 13:08:23 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <4ED31A6C.3000504@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: Oliver Diessel <odiessel@cse.unsw.edu.au>
      | Date: Mon Nov 28 17:21:48 2011
      | Repl: <4ED2DF07.4010207@cse.unsw.edu.au>
      \----------
   /----------
   | M-ID: <4ED2E3A5.6080604@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: WH Wilson <ww@cse.unsw.edu.au>
   | Date: Mon Nov 28 13:28:05 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <65FA592B-80D1-4A89-AA71-BE199AC470C1@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: John Shepherd <jas@cse.unsw.edu.au>
      | Date: Mon Nov 28 13:40:45 2011
      | Repl: <4ED2E3A5.6080604@cse.unsw.edu.au>
      \----------
   /----------
   | M-ID: <1204DCB5-E43D-4AAD-88B9-C5F95016E505@me.com>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Gabriele Keller <gcgk@me.com>
   | Date: Tue Nov 29 11:27:51 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <3D0D4358-0F86-4BC5-B7EB-F806606A5D5A@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Salil Kanhere <sk@cse.unsw.edu.au>
   | Date: Tue Nov 29 13:03:54 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <E8B62DAC-9F40-49E2-BA9F-6DFA70A8D322@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Nadine Marcus <nm@cse.unsw.edu.au>
   | Date: Tue Nov 29 13:19:27 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <Pine.LNX.4.64.1111291922080.7025@kettledrum.orchestra.cse.unsw.EDU.AU>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Meicheng Whale <mcw@cse.unsw.EDU.AU>
   | Date: Tue Nov 29 20:22:52 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <20111129090123.GA7125@cse.unsw.EDU.AU>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Geoff Whale <geoffw@cse.unsw.EDU.AU>
   | Date: Tue Nov 29 21:01:23 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Maurice Pagnucco <mp@cse.unsw.edu.au>
   | Date: Wed Nov 30 01:51:23 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <20111129144226.GB28560@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: Jingling Xue <jx@cse.unsw.edu.au>
      | Date: Wed Nov 30 02:42:26 2011
      | Repl: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
      \----------
      /----------
      | M-ID: <FAD89FCA-B20F-47C9-9036-46FC0E40CA42@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: Malcolm Ryan <mr@cse.unsw.edu.au>
      | Date: Wed Nov 30 11:02:20 2011
      | Repl: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
      \----------
      /----------
      | M-ID: <57428E5F-CC9D-444D-8FAB-CC88249054EA@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: Peter Ho <ph@cse.unsw.edu.au>
      | Date: Wed Nov 30 11:15:09 2011
      | Repl: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
      \----------
      /----------
      | M-ID: <CAD38MSFRVsMtX35P10qcPqs1k1NcYYJVZ5yz2hcfChY4Y1LpHg@mail.gmail.com>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: boualem benatallah <bb@gmail.com>
      | Date: Wed Nov 30 11:39:24 2011
      | Repl: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
      \----------
      /----------
      | M-ID: <Pine.LNX.4.64.1111301118230.14763@cse.unsw.EDU.AU>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: N Paramesh <np@cse.unsw.EDU.AU>
      | Date: Wed Nov 30 12:20:56 2011
      | Repl: <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
      \----------
   /----------
   | M-ID: <20111129175346.GA6258@cse.unsw.EDU.AU>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Albert Nymeyer <an@cse.unsw.EDU.AU>
   | Date: Wed Nov 30 05:53:46 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <20111129211623.GX14327@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Andrew Taylor <at@cse.unsw.edu.au>
   | Date: Wed Nov 30 09:16:23 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <8803EC9B-6EAE-422D-972D-4F02B236F2DB@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Chun Tung Chou <ctc@cse.unsw.edu.au>
   | Date: Wed Nov 30 13:00:39 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <FE4653F7-DAAE-49FF-9A31-0A89E1AF5044@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Salil Kanhere <sk@cse.unsw.edu.au>
   | Date: Wed Nov 30 13:19:21 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
   /----------
   | M-ID: <C7A1685E-2A53-4400-AD25-2DA358A6987F@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: Ken Robinson <kr@cse.unsw.edu.au>
   | Date: Wed Nov 30 15:16:23 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <1BFCE748-18FA-42DA-954F-86BEBF335FAA@cse.unsw.edu.au>
      | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
      | From: Ken Robinson <kr@cse.unsw.edu.au>
      | Date: Wed Nov 30 15:23:32 2011
      | Repl: <C7A1685E-2A53-4400-AD25-2DA358A6987F@cse.unsw.edu.au>
      \----------
   /----------
   | M-ID: <DD2B3A55-13F6-40BC-8615-5F14D4E542AC@cse.unsw.edu.au>
   | Subj: Re: Workshop on Teaching with Reduced Casual Assistance
   | From: sanjay jha <sj@cse.unsw.edu.au>
   | Date: Wed Nov 30 17:55:18 2011
   | Repl: <40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
   \----------
/----------
| M-ID: <21EE5DA5-CCC7-4D78-944B-EAD35B0D8A73@cse.unsw.edu.au>
| Subj: Teaching Workshop
| From: John Shepherd <jas@cse.unsw.edu.au>
| Date: Mon Nov 28 13:15:38 2011
| Repl: (null)
\----------
   /----------
   | M-ID: <290CCAA6-0962-471A-A57D-807AE51C4CC7@cse.unsw.edu.au>
   | Subj: Re: Teaching Workshop
   | From: Eric Martin <em@cse.unsw.edu.au>
   | Date: Mon Nov 28 15:39:38 2011
   | Repl: <21EE5DA5-CCC7-4D78-944B-EAD35B0D8A73@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <CALQ3V4KqG9wL6tX7bAPU5Z6RBCDi7DpA4+PF_KgerNGqg+_HfQ@mail.gmail.com>
      | Subj: Re: Teaching Workshop
      | From: Ashesh Mahidadia <am@cse.unsw.edu.au>
      | Date: Mon Nov 28 15:44:00 2011
      | Repl: <290CCAA6-0962-471A-A57D-807AE51C4CC7@cse.unsw.edu.au>
      \----------
         /----------
         | M-ID: <A844DEED-2655-4E09-8E79-BA53E95DB70C@cse.unsw.edu.au>
         | Subj: Re: Teaching Workshop
         | From: "Richard Buckland (CSE@UNSW)" <rb@cse.unsw.edu.au>
         | Date: Mon Nov 28 17:10:03 2011
         | Repl: <CALQ3V4KqG9wL6tX7bAPU5Z6RBCDi7DpA4+PF_KgerNGqg+_HfQ@mail.gmail.com>
         \----------
            /----------
            | M-ID: <ACDEF6F6-257F-4A6F-8755-2BC9E8BBE134@cse.unsw.edu.au>
            | Subj: Re: Teaching Workshop
            | From: John Shepherd <jas@cse.unsw.edu.au>
            | Date: Mon Nov 28 17:45:17 2011
            | Repl: <A844DEED-2655-4E09-8E79-BA53E95DB70C@cse.unsw.edu.au>
            \----------
               /----------
               | M-ID: <C6CEFFFC-6C43-4FB2-81B7-355850AF0BB2@cse.unsw.edu.au>
               | Subj: Re: Teaching Workshop
               | From: "Richard Buckland (CSE@UNSW)" <rb@cse.unsw.edu.au>
               | Date: Mon Nov 28 19:32:46 2011
               | Repl: <ACDEF6F6-257F-4A6F-8755-2BC9E8BBE134@cse.unsw.edu.au>
               \----------
/----------
| M-ID: <6AB2D6D8-BF45-40F7-BF4A-ACA273179839@cse.unsw.edu.au>
| Subj: Teaching Workshop
| From: John Shepherd <jas@cse.unsw.edu.au>
| Date: Tue Nov 29 23:11:47 2011
| Repl: (null)
\----------
   /----------
   | M-ID: <FF1D31B8-4C4D-4742-B785-60153400289F@cse.unsw.edu.au>
   | Subj: Re: Teaching Workshop
   | From: John Shepherd <jas@cse.unsw.edu.au>
   | Date: Tue Nov 29 23:13:27 2011
   | Repl: <6AB2D6D8-BF45-40F7-BF4A-ACA273179839@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <E05D668D-DE51-4166-9C9B-891688ADB210@cse.unsw.edu.au>
      | Subj: Re: Teaching Workshop
      | From: "Richard Buckland (CSE@UNSW)" <rb@cse.unsw.edu.au>
      | Date: Wed Nov 30 05:45:30 2011
      | Repl: <FF1D31B8-4C4D-4742-B785-60153400289F@cse.unsw.edu.au>
      \----------
   /----------
   | M-ID: <2F6BD270-B773-4EC9-8E14-0AF23E2C2920@cse.unsw.EDU.AU>
   | Subj: Re: Teaching Workshop
   | From: Eric Martin <em@cse.unsw.EDU.AU>
   | Date: Tue Nov 29 23:29:54 2011
   | Repl: <6AB2D6D8-BF45-40F7-BF4A-ACA273179839@cse.unsw.edu.au>
   \----------
      /----------
      | M-ID: <1BD4F884-0713-41C9-AC3F-367EEB8CC0A7@cse.unsw.edu.au>
      | Subj: Re: Teaching Workshop
      | From: John Shepherd <jas@cse.unsw.edu.au>
      | Date: Wed Nov 30 23:29:52 2011
      | Repl: <2F6BD270-B773-4EC9-8E14-0AF23E2C2920@cse.unsw.EDU.AU>
      \----------
         /----------
         | M-ID: <37F601BA-FD32-493E-BDC7-5ABE148C6D69@cse.unsw.edu.au>
         | Subj: Re: Teaching Workshop
         | From: John Shepherd <jas@cse.unsw.edu.au>
         | Date: Thu Dec  1 00:02:48 2011
         | Repl: <1BD4F884-0713-41C9-AC3F-367EEB8CC0A7@cse.unsw.edu.au>
         \----------
            /----------
            | M-ID: <A8BACAB4-084D-424B-8D3C-E6AE65658A9F@cse.unsw.edu.au>
            | Subj: Re: Teaching Workshop
            | From: Eric Martin <em@cse.unsw.edu.au>
            | Date: Thu Dec  1 00:27:34 2011
            | Repl: <37F601BA-FD32-493E-BDC7-5ABE148C6D69@cse.unsw.edu.au>
            \----------
               /----------
               | M-ID: <EA5F7A26-5ADF-4DF3-8203-8991D9E4282A@cse.unsw.edu.au>
               | Subj: Re: Teaching Workshop
               | From: Richard Buckland <rb@cse.unsw.edu.au>
               | Date: Thu Dec  1 08:10:52 2011
               | Repl: <A8BACAB4-084D-424B-8D3C-E6AE65658A9F@cse.unsw.edu.au>
               \----------
            /----------
            | M-ID: <2F3FC28A-3699-4152-8D12-98B458010EC2@cse.unsw.edu.au>
            | Subj: Re: Teaching Workshop
            | From: Richard Buckland <rb@cse.unsw.edu.au>
            | Date: Thu Dec  1 08:25:11 2011
            | Repl: <37F601BA-FD32-493E-BDC7-5ABE148C6D69@cse.unsw.edu.au>
            \----------
         /----------
         | M-ID: <99EAAD38-7A42-4C31-98A9-4267E4794E20@cse.unsw.edu.au>
         | Subj: Re: Teaching Workshop
         | From: Eric Martin <em@cse.unsw.edu.au>
         | Date: Thu Dec  1 00:26:11 2011
         | Repl: <1BD4F884-0713-41C9-AC3F-367EEB8CC0A7@cse.unsw.edu.au>
         \----------
            /----------
            | M-ID: <B049B2B1-D4C0-4CD2-A0AD-43513A9D920B@cse.unsw.edu.au>
            | Subj: Re: Teaching Workshop
            | From: Richard Buckland <rb@cse.unsw.edu.au>
            | Date: Thu Dec  1 08:35:25 2011
            | Repl: <99EAAD38-7A42-4C31-98A9-4267E4794E20@cse.unsw.edu.au>
            \----------
         /----------
         | M-ID: <1C13A30E-64F8-4B70-A2C5-E4D56DAEC699@cse.unsw.edu.au>
         | Subj: Re: C.R.A.P Workshop
         | From: Richard Buckland <rb@cse.unsw.edu.au>
         | Date: Thu Dec  1 08:22:29 2011
         | Repl: <1BD4F884-0713-41C9-AC3F-367EEB8CC0A7@cse.unsw.edu.au>
         \----------
/----------
| M-ID: <95367193-11B5-48E3-B929-421A7BC7F138@cse.unsw.edu.au>
| Subj: Follow-up to Teaching Workshop
| From: John Shepherd <jas@cse.unsw.edu.au>
| Date: Fri Dec  2 10:55:52 2011
| Repl: (null)
\----------
